Polyfills/Window/Source/Window.cpp

```diff
@@ -10,18 +11,23 @@ namespace Babylon::Polyfills::Internal
         constexpr auto JS_CLASS_NAME = "Window";
         constexpr auto JS_SET_TIMEOUT_NAME = "setTimeout";
         constexpr auto JS_A_TO_B_NAME = "atob";
+
         constexpr auto JS_ADD_EVENT_LISTENER_NAME = "addEventListener";
         constexpr auto JS_REMOVE_EVENT_LISTENER_NAME = "removeEventListener";
+        auto JS_DEVICE_PIXEL_RATIO_NAME = "devicePixelRatio";
```

Comment:
```suggestion
        constexpr auto JS_DEVICE_PIXEL_RATIO_NAME = "devicePixelRatio";
```

Reply:
Thanks,  I must have changed this trying to fix my JS wrapper.